translatorbot/.env.example

```diff
@@ -0,0 +1,32 @@
+TTS_SERVICE=azure # or google
+LLM_SERVICE=azure # or openai
+IMAGE_GEN_SERVICE=azure # or openai
+
+# Your Daily API key (to create new rooms and meeting tokens)
+DAILY_API_KEY=
+
+# Needed for Azure Speech
+AZURE_SPEECH_SERVICE_REGION=westus
+AZURE_SPEECH_SERVICE_KEY=
+
+#Needed for Azure LLM (ChatGPT)
+AZURE_CHATGPT_KEY=
+AZURE_CHATGPT_ENDPOINT=
+AZURE_CHATGPT_DEPLOYMENT_ID=
+
+# Needed for Azure DALL-E
+AZURE_DALLE_KEY=
```

Comment:
Does this translation demo use image generation, or can some of these env variables be removed?